### ISSUE TITLE
Add trace_chain to shortener_keys defaults

### ISSUE THREAD

**Issue by bisognano** [rollbar/pyrollbar#414]
Hi! I was running into an issue wherein I was sending my locals to rollbar but getting the error
```
Failsafe from pyrollbar: payload too large. Original payload may be found in your server logs by searching for the UUID.
```
The solution was to add a new `shortener_key` to the config as 
```
[("body", "trace_chain", "*", "frames", "*", "locals", "*")]
```
Looking at the defaults for `shortener_keys` for when `locals.enabled` is `true`, I would have anticipated `trace_chain` locals handling here as well.
```
[ ('body', 'trace', 'frames', '*', 'code'), ('body', 'trace', 'frames', '*', 'args', '*'), ('body', 'trace', 'frames', '*', 'kwargs', '*'), ('body', 'trace', 'frames', '*', 'locals', '*') ]
```

Would it be reasonable to add that `trace_chain` tuple to the defaults on `shortener_chain` as well? Looks like theres a stagnant open PR for this currently - https://github.com/rollbar/pyrollbar/pull/365

**Comment by danielmorell** [comment]
Hi @bisognano, sorry for the delay on this. I think this sounds like a reasonable default. I will take a look at #365.

**Comment by danielmorell** [comment]
This was resolved when #365 was merged.
